BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/de.lproj/Localizable.strings

```diff
@@ -85,6 +85,8 @@
 "ginibank.digitalinvoice.edit.name.error" = "Bitte geben Sie den Artikelnamen ein.";
 "ginibank.digitalinvoice.edit.price.error" = "Bitte geben Sie den Artikelpreis ein.";
 
+"ginibank.digitalinvoice.skonto.title" = "Skonto";
```

Comment:
[clarification] Why we can not use the normal skonto title? Do we really need a new one in Digital invoice? I think this is adding extra work for customers. I would say it's ok to reuse from skonto since is same feature.

Reply:
We can use "ginibank.skonto.screen.title" string, I just added new one if its needed to be customized. If its not needed, I prefer also to reuse existing one